tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -105,28 +107,42 @@ public:
 /**
 \class ROOT::Experimental::RNTupleModel
 \ingroup NTuple
-\brief The RNTupleModel encapulates the schema of an ntuple.
+\brief The RNTupleModel encapulates the schema of a RNTuple.
 
 The ntuple model comprises a collection of hierarchically organized fields. From a model, "entries"
-can be extracted. For convenience, the model provides a default entry unless it is created as a "bare model".
-Models have a unique model identifier that faciliates checking whether entries are compatible with it
+can be extracted or created. For convenience, the RNTupleModel provides a default entry unless it is created as a "bare model".
+Models have a unique model identifier that facilitates checking whether entries are compatible with it
 (i.e.: have been extracted from that model).
 
-A model is subject to a state transition during its lifetime: it starts in a building state, in which fields can be
-added and modified.  Once the schema is finalized, the model gets frozen.  Only frozen models can create entries.
-From frozen, models move into a expired state.  In this state, the model is only partially usable: it can be cloned
+A model is subject to state transitions during its lifetime: it starts in a Building state, in which fields can be
+added and modified.  Once the schema is finalized, the model gets Frozen.  Only frozen models can create entries.
+From frozen, models move into an Expired state. In this state, the model is only partially usable: it can be cloned
```

Comment:
Not sure if I agree with the capitalization here, I think together with the diagram below it reads quite naturally as is. Perhaps use something else to emphasize the different states:
```suggestion
A model is subject to state transitions during its lifetime: it starts in a *building* state, in which fields can be
added and modified.  Once the schema is finalized, the model gets *frozen*.  Only frozen models can create entries.
From frozen, models move into an *expired* state. In this state, the model is only partially usable: it can be cloned
```